mautic/mautic
# Small text update to reflect change from apply to save on button

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y ]
| Deprecations?                          | [n ]
| BC breaks? (use the c.x branch)        | [n ]
| Automated tests included?              | [ n/a] 

#### Description:
I noticed that when we show a message when setting up a mail transport advising the user that they must first commit their changes before sending a test email, we use the old term 'apply' which the button used to say, whereas now it says 'save'.

I think we should therefore update the message to ensure it does reflect what the button says.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Set up a mail transport (e.g. Sendgrid API) and notice the message which appears - this PR will change 'apply' to 'save'


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10826"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
